### PR TITLE
2021 03 26 adelacecilia html coding session number four

### PR DESCRIPTION
**Pull request by adelacecilia**

### CODE REVIEW

**Review by robertoalx — APPROVED** [review]
💯 👍

File: web/adelacecilia/html/meter tag/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>The Meter tag</title>
+  </head>
+  <body>
+    <p>
+      Below is a meter that meansure how many people will go to HEAVEN and how
```
Comment (QuintyHH) [inline web/adelacecilia/html/meter tag/index.html]: Tiny typo here, it's 'measures', but not a breaking change.

File: web/adelacecilia/html/nav tag/index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <title>The nav tag</title>
+  <body>
+    <h1>The nav element</h1>
+
+    <p>Social media LINKS:</p>
+
+    <nav>
```
Comment (QuintyHH) [inline web/adelacecilia/html/nav tag/index.html:9]: No need to leave empty lines in html.

**Review by QuintyHH — APPROVED** [review]
2 small changes, no need for a CR, but please be careful next time :)

**Review by biggicode — APPROVED** [review]

**Review by RalucaRbc — APPROVED** [review]
💯

File: web/adelacecilia/html/nav tag/index.html (hunk 2 above)
Comment (adelacecilia) [reply web/adelacecilia/html/nav tag/index.html:9]: Prittier is doing that  :))

**Review by simonamarcu2 — APPROVED** [review]
Awesome

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**